requirements.txt

```diff
@@ -8,3 +8,4 @@ Pygments~=2.1
 PyPrint~=0.2.6
 requests~=2.12
 setuptools>=17.0
+attrs~=16.3.0 
```

Comment:
could you place it in the alphabetically right position? makes it easier to find ;)